core/include/kernel/huk_subkey.h

```diff
@@ -12,9 +12,10 @@
 
 /*
  * enum huk_subkey_usage - subkey usage identifier
- * @HUK_SUBKEY_RPMB:	RPMB key
- * @HUK_SUBKEY_SSK:	Secure Storage key
- * @HUK_SUBKEY_DIE_ID:	Representing the die ID
+ * @HUK_SUBKEY_RPMB:	  RPMB key
```

Comment:
I would have preferred if the changes in this file went in a separate commit.

Reply:
Sure, I can do that.